app.js

```diff
@@ -0,0 +1,41 @@
+//jshint esversion:6
+const express = require('express');
+const exphbs = require ('express-handlebars');
+const bodyParser = require('body-parser');
+const methodOverride = require ('method-override');
+//const articlesRoute = require('./routes/articles.js');
+const productsRoute = require('./routes/products.js');
+const PORT = process.env.PORT|| 8080;
+const app = express();
+
+//TO MAKE SURE THAT IT RECOGNIZES THE HBS EXT
+app.engine('.hbs', exphbs({
+  defaultLayout: 'main',
+  extname: '.hbs'
+}));
+app.set('view engine', '.hbs');
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(methodOverride('_method'));
+
+
+app.use(methodOverride((req,res) => {
```

Comment:
You don't need to make a custom method to handle the PUT and DELETE http methods. There's a simpler way: 

``` javascript
  app.use(methodOverride('_method'))
```

It looks like you already have that configured as well so you can just remove this section. You just need one of these things, not both.